calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
```

Comment:
don't think this ever gets called. When I start the node with `--backend=gobgp` and get the `status` it still tries to connect to BIRD and fails. See the output below:

```bash

vagrant@calico-01:~/calico-containers/dist$ sudo ./calicoctl node --node-image=calico/node --no-pull --backend=gobgp
Running Docker container with the following command:

docker run -d --restart=always --net=host --privileged --name=calico-node -e HOSTNAME=calico-01 -e IP= -e IP6= -e CALICO_NETWORKING_BACKEND=gobgp -e AS= -e NO_DEFAULT_POOLS= -e ETCD_AUTHORITY=172.17.8.101:2379 -e ETCD_SCHEME=http -v /var/log/calico:/var/log/calico -v /lib/modules:/lib/modules -v /var/run/calico:/var/run/calico calico/node

Calico node is running with id: f8150715ef6e1740276e33eb60acae6cad36365d3260187fce04714a3864e43a
Waiting for successful startup
No IP provided. Using detected IP: 172.17.8.101
CALICO_NETWORKING_BACKEND is gobgp - run calico-bgp-daemon
Calico node started successfully

vagrant@calico-01:~/calico-containers/dist$ sudo ./calicoctl status
calico-node container is running. Status: Up About a minute
Running felix version 1.4.2.dev1

IPv4 BGP status
IP: 172.17.8.101    AS Number: 64511 (inherited)
Unable to connect to server control socket (/var/run/calico/bird.ctl): Connection refused
sh: echo: I/O error
Couldn't connect to bird.
IPv6 BGP status
No IPv6 address configured.
```

Maybe I'm not doing it right? If so can you share your test results?